pkg/game/field/field.go

```diff
@@ -119,16 +132,51 @@ func (field *Field) Expand(board elements.Board, cityManager city.Manager) {
 	field.features = newFeatures
 }
 
+// Returns true if the field has lesser or equal number of meeples than maxMeepleCount, and false otherwise
+func (field Field) IsFieldValid(board elements.Board, maxMeepleCount int) bool {
```

Comment:
a) it should be documented that this method cannot be used twice since it alters the `field`'s state (maps are pointers so they point to the same map on copy)
b) it would be easier to realise that this can modify the field if it were using a pointer receiver

Reply:
changed so that it doesn't modify the original field.features, and instead works on a copy